src/systemd_ctypes/event.py

```diff
@@ -100,7 +101,9 @@ def run_async(main):
 
         asyncio._systemd_ctypes_polyfills = True
 
-    asyncio.run(main)
+        # this does not support the debug argument, ignore it
+        asyncio.run(main)
```

Comment:
Instead of doing this, let's replace asyncio.run with a function which does take the kwarg, and ignores it.  The idea here is to install compatible definitions, after all.

Reply:
I checked what Python 3.11 actually does with this arg.  It calls `.set_debug()` on the loop.  That function is also in Python 3.6, so I guess that's the right thing to do here.

Reply:
Right, done. I gave up on my previous attempt at not actually defining asyncio.run(). The patch is now much less intrusive.